aspects/aspects.go

```diff
@@ -504,7 +524,7 @@ func get(subKeys []string, index int, node map[string]json.RawMessage, result in
 	// decode the next map level
 	var level map[string]json.RawMessage
 	if err := jsonutil.DecodeWithNumber(bytes.NewReader(rawLevel), &level); err != nil {
-		// TODO see TODO in NewAspectBundle
+		// TODO: see TODO in NewAspectBundle
```

Comment:
TODO in newAspect?

Reply:
Yes, it changed since this comment was written. Thanks